sdk/resourcemanager/loadtestservice/armloadtestservice/go.mod

```
@@ -1,3 +1,4 @@
+// Deprecated: use github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/loadtesting/armloadtesting instead.
```

Could we add a `please` here?
```suggestion
// Deprecated: please use github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/loadtesting/armloadtesting instead.
```